src/python/nimbusml.pyproj

```diff
@@ -545,7 +537,9 @@
     <Compile Include="nimbusml\preprocessing\normalization\minmaxscaler.py" />
     <Compile Include="nimbusml\preprocessing\normalization\__init__.py" />
     <Compile Include="nimbusml\preprocessing\schema\columnconcatenator.py" />
-    <Compile Include="nimbusml\preprocessing\schema\columndropper.py" />
+    <Compile Include="nimbusml\preprocessing\schema\columndropper.py">
+      <SubType>Code</SubType>
+    </Compile>
     <Compile Include="nimbusml\preprocessing\schema\columnduplicator.py" />
```

Comment:
Remove this, so its same as other lines

Reply:
done


---
In reply to: [248141903](https://github.com/Microsoft/NimbusML/pull/77#discussion_r248141903) [](ancestors = 248141903)